hooks/post/30-upgrade.rb

```diff
@@ -137,6 +138,24 @@ def remove_event_queue
   end
 end
 
+def remove_legacy_mongo
+    logger.info 'Verifying if Mongo 3.x is running'
+    # Start mongo if not running
+    unless Kafo::Helpers.execute('pgrep mongod')
+      Kafo::Helpers.execute('service-wait rh-mongodb34-mongod start')
+    end
```

Comment:
Just curious why this check is needed? Given the installer should start it if it reached this point.